Dockerfile

```diff
@@ -27,6 +27,8 @@ RUN pip install build
 RUN apt-get update && \
     apt-get install -y --no-install-recommends git
 COPY . /src/fmriprep
+# Ensure clean build
+RUN git -C /src/fmriprep reset --hard && git -C /src/fmriprep clean -fxd
```

Comment:
This makes our build too rigid - we should still allow building with uncommitted changes